Algorithm.Python/BasicTemplateOptionsHistoryAlgorithm.py

```diff
@@ -69,7 +69,7 @@ def OnData(self,slice):
                     contract.ImpliedVolatility))
 
     def OnSecuritiesChanged(self, changes):
-        if changes == SecurityChanges.None: return
+        if changes == None: return
```

Comment:
Does this evaluate properly? We could add `SecurityChanges.Count` and instead check `changes.Count == 0` ?

Reply:
I think we could remove this one here, because if `changes == SecurityChanges.None`, it will loop throw empty lists below, but `changes.Count == 0` is a good solution.